### PR TITLE
Pylint cleanup

### PR DESCRIPTION
**Pull request by carsongee**
Just a bunch of pylint issues that have piled up for cases where pyflakes is nicer.  There are still several violations I have left, but I think #24 will fix most of the names that are too long and the too-many-methods in test_github.py.

### CODE REVIEW

File: orcoursetrion/tests/base.py [hunk 1]
```diff
@@ -96,11 +101,16 @@ def callback_team_list(
...
             self, request, uri, headers,
-            status_code=200, members=TEST_TEAM_MEMBERS
+            status_code=200, members=None
     ):
         """
         Return team membership list
         """
+        # Disabling unused-argument because this is a callback with
+        # required method signature.
+        # pylint: disable=unused-argument,too-many-arguments
```
Comment (bdero) [inline orcoursetrion/tests/base.py]: Does this pylint comment actually trickle down to the correct statement (the assert)?

Comment (carsongee) [inline orcoursetrion/tests/base.py]: It is trickling up to the function which has too many arguments, and they aren't all used. The assert should be fine

Comment (bdero) [inline orcoursetrion/tests/base.py]: Ah, my bad.

**Comment by bdero** [comment]
:electric_plug:

**Pull request merged**